corehq/apps/users/management/commands/gdpr_scrub_user_from_forms.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import unicode_literals
+from django.core.management.base import BaseCommand
+from corehq.apps.users.models import CommCareUser
+from corehq.form_processor.interfaces.dbaccessors import FormAccessors
+from io import StringIO
+from lxml import etree
+import sys
+import six
+import logging
+
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = "Scrubs the username from all forms associated with the given user"
+
+    def add_arguments(self, parser):
+        parser.add_argument('username')
+        parser.add_argument('domain')
+
+    def handle(self, username, domain, **options):
+        this_form_accessor = FormAccessors(domain=domain)
+        user = CommCareUser.get_by_username(username)
+        if not user:
+            logger.info("User {} not found.".format(username))
+            sys.exit(1)
+        user_id = user._id
+        form_ids = this_form_accessor.get_form_ids_for_user(user_id)
+        new_username = "Redacted User (GDPR)"
```

Comment:
might be nice to have this as a constant

Reply:
Done here:
https://github.com/dimagi/commcare-hq/pull/19979/commits/9782b6ada66f767615c35af61d31483a9a562b70